internal/resources/policy/type/custom/constants.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright © 2022 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package custom
+
+import reciperesource "github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/policy/type/custom/recipe"
+
+const (
+	ResourceName              = "tanzu-mission-control_custom_policy"
+	nameKey                   = "name"
+	specKey                   = "spec"
+	inputKey                  = "input"
+	typeDefaultValue          = "custom-policy"
+	recipeVersionDefaultValue = "v1"
+)
+
+// Allowed input recipes.
+const (
+	unknownRecipe                     recipe = ""
```

Comment:
Can we declare it in the policy package and reuse it for all the policies?

as in 

`unknownRecipe recipe = policy.unknownRecipe`

and in the policy package, we can declare it as `""` just to ensure we are not initializing this differently everywhere.